continuous_integration/home/coder/.local/bin/build-legate

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+
+build_legate_all() {
+    build-legate-cpp;
+    build-legate-wheel;
+    build-legate-conda;
+}
+
+build_legate_release() {
+    . conda-utils
+    activate_conda_env
+    mkdir -p /tmp/env_yaml /tmp/conda-build
+    conda env export > /tmp/env_yaml/pre_build.yml
```

Comment:
What is this for? Same for the next one.

Reply:
Good catch, we don't need this.